iset.mm

```diff
@@ -39754,6 +39754,7 @@ The natural numbers (i.e. finite ordinals)
       cint sbimi clelsb4 sylib sylbi mpgbir dfom3 eleqtrri ) DDAEZFZBEGUFFBUFHZ
       IZAJZRZKDUKFCEZUJFZDULFZLCCDUJMNUMUIACOZUNUICAPUOUGACOUNUIUGACUGUHQSCADTU
       AUBUCABUDUE $.
+    $( $j usage 'peano1' avoids 'ax-iinf' 'ax-inf2' 'ax-setind'; $)
```

Comment:
ax-inf2 is not even in main.  Why do we want to have it mentioned here?

Reply:
Ah, I didn't notice that was from a mathbox and is different than `ax-inf2` in set.mm. Probably we should rename it to avoid that confusion between the iset.mm one and the set.mm.

(And I agree there is no need for either it or https://us.metamath.org/ileuni/ax-infvn.html to be mentioned in the $j usage commands in main).

Reply:
ax-inf2 removed.